COCOAPODS.md

```diff
@@ -1,150 +1,343 @@
 # CocoaPods integration
 
-Starting with 1.3.30, an experimental integration with [CocoaPods](https://cocoapods.org/) is added
-to Kotlin/Native. This feature allows you to represent a Kotlin/Native Gradle-project as a
-CocoaPods dependency. Such a representation provides the following advantages:
+Kotlin/Native provides integration with the [CocoaPods dependency manager](https://cocoapods.org/). You can add
+dependencies on Pod libraries stored in the CocoaPods repository or locally as well as use a Kotlin/Native Gradle
+project as a CocoaPods dependency (Kotlin Pod).
 
- - Such a dependency can be included in a Podfile of an Xcode project and automatically built (and rebuilt)
- along with this project. As a result, importing to Xcode is simplified since there is no need to
- write corresponding Gradle tasks and Xcode build steps manually.
+You can manage Pod dependencies right in Intellij IDEA enjoying its benefits of working with code, such as code
+ highlighting and completion. You can build the whole Kotlin project with Gradle as a whole not switching to Xcode. 
+ Go to Xcode only when you need to write Swift/Objective-C code or run your application on a simulator or device.
  
- - When building from Xcode, you can use CocoaPods libraries without writing
- .def files manually and setting cinterop tool parameters. In this case, all required parameters can be
- obtained from the Xcode project configured by CocoaPods.
+Depending on your purpose, you can add dependencies between:
+* [A Kotlin project and a Pod library from the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository)
+* [A Kotlin project and an Objective-C Pod library stored locally](#add-a-dependency-on-an-objective-c-pod-library-stored-locally)
+* [A Kotlin Pod and an Xcode project with one target](#add-a-dependency-between-a-kotlin-pod-and-xcode-project-with-one-target) 
+or [several targets](#add-a-dependency-between-a-kotlin-pod-with-an-xcode-project-with-several-targets)
+* [Several Kotlin Pods and an Xcode project](#add-dependencies-between-multiple-kotlin-pods-and-an-xcode-project)
```

Comment:
There is no sections for multiple Xcode projects. We should at least mention whether it is supported.

Reply:
Current state of Cocoapods Integration doesn't support multiple Xcode projects.

Reply:
We cannot automatically call `pod install` in this case. But AFAIK a user can depend on the same Kotlin pod from multiple Xcode projects given that he calls `pod install` manually for each of these projects. Is't this the case?

Reply:
Of course yes, this case may be supported by manual `pod install` calls.

Reply:
Then it worth mentioning because currently users may think that this case is fully unsupported.

Reply:
I can add a note about this without providing a procedure. Does it work?

Reply:
I believe a note is enough here.

Reply:
Added a note after the list of cases. @SvyatoslavScherbina and @ilmat192 , please take a look if it's ok.

Reply:
It looks good to me.

Reply:
It looks good to me.

Reply:
👍